### PR TITLE
Updates some methodology text

### PR DESCRIPTION
**Pull request by joshzarrabi**
## Preview Link

<a href="https://deploy-preview-1803--health-equity-tracker.netlify.app/methodology" target="_blank">
  <br />
Click to demo updates </a> 

## Description

* Updates some text about our process
* Clarifies why we use NYTimes COVID data as a comparison source
* Updates vaccination data limitations.

## Motivation and Context

In services of #1706

## How has this been tested?

Locally

## Types of changes

- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Review by eriwarr — APPROVED** [review]
👍🏿

File: frontend/src/pages/WhatIsHealthEquity/FaqTab.tsx [hunk 1]
```diff
@@ -107,23 +107,19 @@ Specific information on update frequencies by source can be found on our <a href
     a: `
   <ul>
     <li>
-    In an effort to be fully transparent, all data is retrieved from publicly sourced APIs and manual downloads
+      All data presented in the Health Equity Tracker are retrieved from publicly sourced application programming interfaces (APIs) and manual downloads. The data extracted from these sources are continuously updated to retrieve the most relevant data.
```
Comment (benhammondmusic) [inline frontend/src/pages/WhatIsHealthEquity/FaqTab.tsx:110]: I'd rephrase the last sentence:

"These sources are continuously updated to extract the most relevant data."

**Review by benhammondmusic — APPROVED** [review]
Great
